tests/test_local_move.py

```diff
@@ -64,6 +64,14 @@ def assert_no_drift(
 
     expected_selection_fraction_traj = np.array([observable_fxn(x) for x in traj])
 
+    # A sanity check that the early samples don't have a massive jump within
+    # in the case of unstable local MD this test can pass neighboring atoms go from ~10% of the system to ~100% of the system
+    # in the first step
+    differences_early = np.diff(expected_selection_fraction_traj[:n_samples]) / expected_selection_fraction_traj[0]
+    assert (
+        np.any(differences_early) < 2.0
```

Comment:
This doesn't seem correct?

Reply:
Fixed in [282a845](https://github.com/proteneer/timemachine/pull/1014/commits/282a84539b4b34e2168eb87c0fcaaef4edfe8e75)